admincli/cli.go

```diff
@@ -137,8 +152,8 @@ func GetContentFromDir(content string, dirPath string) string {
 	return content
 }
 
-func GenerateFile(table string, db *mysql.Mysql) {
-	columnsModel, _ := db.Query("show columns in " + table)
+func GenerateFile(table string, conn db.Connection, fieldField, typeField string) {
+	columnsModel, _ := conn.Query("show columns in '" + table + "'")
```

Comment:
@theromis It doesn't seem to work when using mysql.

Reply:
@chenhg5 then probably better to have some function "ShowColumns" for every DB.
for mysql it will be `show columns in my_table_name` in postgresql it will be `select column_name, udt_name from information_schema.columns where table_name = 'my_table_name'` please advise.

Reply:
@theromis yes, i am on it

Reply:
@chenhg5 pushed ShowColumns function, please check

Reply:
@theromis Thx. I add a new db module named dialect for different sql syntactic in the several commits just now to the master branch. And you can test and modify it on this basis.